data_management/models.py

```diff
@@ -623,7 +624,9 @@ class ExternalObject(BaseModel):
       modelling pipeline.***
 
     ### Writable Fields:
-    `doi_or_unique_name`: DOI or Name of the `ExternalObject`, unique in the context of the (`doi_or_unique_name`, `title`)
+    `identifier` (*optional*): Full URL of identifier (e.g. DataCite DOI) of the `ExternalObject`
+
+    `other_unique_name`: Name of the `ExternalObject`, unique in the context of the (`other_unique_name`, `title`)
```

Comment:
These should be alternatives presumably rather than `identifier` being optional? And both unique in the context of the (`xxx`, `title`, `version`).

Reply:
Improved in https://github.com/FAIRDataPipeline/data-registry/pull/92/commits/be69d2a9ccec7b398121c4d5bcad202e32364b30

Reply:
Fixed in https://github.com/FAIRDataPipeline/data-registry/commit/be69d2a9ccec7b398121c4d5bcad202e32364b30